prismic-configuration.json

```diff
@@ -0,0 +1,3 @@
+{
+    "apiEndpoint": "https://your-repo-name.cdn.prismic.io/api/v2"
```

Comment:
Why was this added? What is it doing?

Reply:
I'm. not exactly sure if the theme looks for that key name: `apiEndpoint` of the value `your-repo-name`, but I added it this way because that's the way we do it in most cases, using a .json or .js file.

Reply:
n

Reply:
I'm. not exactly sure if the theme looks for that key name: `apiEndpoint` of the value `your-repo-name`, but I added it this way because that's how we do it in most cases, using a .json or .js file.

You'd still need to update the config.php file manually, but that's something I'll mention in the guide.

Reply:
I'm not exactly sure if the theme looks for the key name `apiEndpoint` of the value `your-repo-name` or both, but I added it this way because that's how we do it in most cases, using a .json or .js file, [as explained here](https://user-guides.prismic.io/en/articles/764581-how-to-create-a-theme-to-quickstart-your-projects-for-prismic)

You'd still need to update the config.php file manually, but that's something I'll mention in the guide.

Reply:
@PaulinaVPG Okay so you just added this file. It doesn't actually do anything then. You can remove it.

You could have always asked about how the `theme` command works ;)

It will look for `your-repo-name` and replace that in the file you specify with `--conf` when running the theme command.

So you can get rid of this and just make sure to add the `--conf` to the theme command and specify the config.php file.

Reply:
k, file deleted !